tox.ini

```diff
@@ -18,6 +18,7 @@ envlist =
 [testenv]
 deps =
     setuptools  <74
+    git+https://github.com/zopefoundation/zope.proxy@62c4a3ba359b30fc20facc1f8f9482975ee22b60#egg=zope.proxy
```

Comment:
Are we sure we want tests done with a fixed version of `zope.proxy`?

I understand that you had a need to make tests against that fixed version to analyze the observed test failures (which you now found to have been caused by the released `zope.proxy` version to interpret `PURE_PYTHON` still in the "old" way), but please do not forget to remove this test version pin again once a new `zope.proxy` is released.

Reply:
You are right, this is only to run the tests before new `zope.proxy` is released. This pull request is still draft, once the new `zope.proxy` is released I will revert this change and mark it ready for review. 

I was thinking we can proceed with the steps described in https://github.com/zopefoundation/zope.proxy/pull/71#issuecomment-2387483569